stats/info_statistics.py

```diff
@@ -267,7 +267,9 @@ def get_smart_answers(self):
         logger.info('Getting smart answers')
 
         smart_answers = []
-        url = 'https://www.gov.uk/api/search.json?filter_format=smart-answer'
+        url = 'https://www.gov.uk/api/search.json'
+        url += '?filter_format=smart-answer'
+        url += '&filter_format=simple_smart_answer'
```

Comment:
@rboulton if I wish the Search API to return all smart answers AND all simple smart answers, this is the right way to do it?